catkin_tools_document/doxygen.py

```diff
@@ -67,7 +67,8 @@ def generate_doxygen_config(logger, event_queue, conf, package, recursive_build_
         'PROJECT_NAME': package.name,
         'OUTPUT_DIRECTORY': output_path,
         'TAB_SIZE': conf.get('tab_size', '8'),
-        'TAGFILES': ' '.join(tagfiles)
+        'TAGFILES': ' '.join(tagfiles),
+        'USE_MATHJAX': True
```

Comment:
@mikepurvis If you want, I can allow to pass a configuration option from `conf.py`, doing:
`'USE_MATHJAX': conf.get('use_mathjax', 'True')`

And I can default to `False` if you think it's important to keep previous behaviour.